### ISSUE TITLE
Field not found on tracked model when PGHISTORY_DEFAULT_TRACKERS is set and more than two models are tracked with default trackers.

### ISSUE THREAD

**Issue by enoch2110** [Opus10/django-pghistory#110]
There is a strange behavior when `PGHISTORY_DEFAULT_TRACKERS` is set.

When `PGHISTORY_DEFAULT_TRACKERS` is set in `settings.py` and more than two models are tracked with default trackers, pgtrigger tries to find a field on the model that exists on the other model.

Provided the following,

settings.py
```python
PGHISTORY_DEFAULT_TRACKERS = (pghistory.InsertEvent(), pghistory.UpdateEvent())
```

app/models.py
```python
@pghistory.track()
class ModelA(models.Model):
    name_a = models.CharField()


@pghistory.track()
class ModelB(models.Model):
    name_b = models.CharField()
```

When running `makemigrations` or `pgtrigger install`, following error occurs.

```
ValueError: Field "name_a" not found on model "<class 'play.models.ModelB'>"
```

pgtrigger tries to find a field that exists on ModelA on ModelB.

Such an error does not occur when trackers are provided in `pghistory.track` or `PGHISTORY_DEFAULT_TRACKERS` is not set.

I tested on newly created django project and the package versions were as follows.

- django-pghistory: v3.1.0
- django-pgtrigger: v4.11.0
- django: v5.0
- psycopg2-binary: v2.9.9
- PostgreSQL: v15.3

By the way, I really enjoy using pgtrigger and pghistory. Thank you for maintaining such a great library.

**Comment by foobarna** [comment]
I've started adding `django-pghistory` to the project and I've also encountered this behaviour where `PGHISTORY_DEFAULT_TRACKERS = (pghistory.InsertEvent(), pghistory.UpdateEvent(), pghistory.DeleteEvent())` and `@pghistory.track(pghistory.InsertEvent(), pghistory.UpdateEvent(), pghistory.DeleteEvent())` does not behave the same. The former errors out with the same message as in description.

My first assumption was that it errors out on fields with foreign key where the name does not equal db_column, ie `store_rep = models.ForeignKey(Store, db_column='store_id', on_delete=models.PROTECT)`.

- PSQL v14
- Django==4.2.7
- psycopg2==2.9.6
- django-pghistory~=3.1.0

**Comment by SupImDos** [comment]
I've run into this issue as well. My assumption is that `pghistory.Tracker` instances aren't designed to be re-used across multiple models, so I've created a PR (#133) that ensures the provided `PGHISTORY_DEFAULT_TRACKERS` are copied before usage.

**Comment by wesleykendall** [comment]
this is resolved in v3.3.0
